dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/eols/NewConditionTransformations.scala

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.monster.dap.eols
+
+import org.broadinstitute.monster.dap.common.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.EolsNewCondition
+
+object NewConditionTransformations {
+
+  def mapNewConditionMetadata(rawRecord: RawRecord): EolsNewCondition = {
+    //    val ToxinConsumption = Some(diagnosisType.contains("2"))
```

Comment:
Eric and I were trying to think about whether there is a quick way to make this code a little more DRY but I don't know if it would be worth the effort given the number of fields in this fragment. I feel like there are a few instances throughout DAP where a list of items is mapped to specific integers and those integers/item prefixes are reused in follow up questions. In particular I thought of the Activity Values in [DemographicsTransformations](https://github.com/DataBiosphere/dog-aging-ingest/blob/master/dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformations.scala#L243

Reply:
Yeah, we could whip up a small DSL for this potentially, and then codegen these transformation classes. @quazi-broad can you ticket?

Reply:
@quazi-broad another thought occurs, this would be a good coding jamboree project.